iModelCore/ECPresentation/Tests/NonPublished/Integration/Content/ContentIntegrationTests.cpp

```diff
@@ -5735,6 +5735,230 @@ TEST_F(RulesDrivenECPresentationManagerContentTests, ContentInstancesOfSpecificC
     EXPECT_STREQ("Derived2-222", jsonValues2["CalculatedProperty_0"].GetString());
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsitest
++---------------+---------------+---------------+---------------+---------------+------*/
+DEFINE_SCHEMA(ContentInstancesOfSpecificClassesSpecification_ContentModifierAppliesCalculatedPropertiesOnNestedContentProperties, R"*(
+    <ECEntityClass typeName="A">
+        <ECNavigationProperty propertyName="B" relationshipName="A_Has_B" direction="Forward" />
+        <ECNavigationProperty propertyName="C" relationshipName="A_Has_C" direction="Forward" />

```

Comment:
any specific reason you need those navigation props?

Reply:
deleted